src/Config/ConfigPanel.ts

```diff
@@ -88,10 +94,33 @@ export class ConfigPanel {
   }
 
   private _getHtmlForWebview(webview: vscode.Webview, context: vscode.ExtensionContext) {
-    // TODO Use a nonce to only allow specific scripts to be run
-    // TDOO fill with logics related to html
-    // TODO import html and change html source to fit webview format
-
-    return '';
+    const scriptUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'index.js'));
+
+    // Uri to load styles into webview
+    const stylesResetUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'reset.css'));
```

Comment:
how about adding a method for `vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'reset.css')` ?
something like,
```js
private getMediaPath(file: string) {
   return vscode.Uri.joinPath(this._extensionUri, 'media/Config', file);
}
```
- there exist two patterns that can be reduced
- long line will become shorter which will enhance readability

Reply:
Please take this as optional, we can refactor this later.